lib/matplotlib/layout_engine.py

```diff
@@ -10,9 +10,14 @@
 layout engine while the figure is being created.  In particular, colorbars are
 made differently with different layout engines (for historical reasons).
 
-Matplotlib supplies two layout engines, `.TightLayoutEngine` and
-`.ConstrainedLayoutEngine`.  Third parties can create their own layout engine
-by subclassing `.LayoutEngine`.
+Matplotlib has two built-in two layout engines:
```

Comment:
```suggestion
Matplotlib has two built-in layout engines:
```
(Assuming the second `two` was a typo.)